src/Form/Field.php

```diff
@@ -207,16 +207,19 @@ public static function defaults(): array
             ],
             'computed' => [
                 'after' => function () {
+                    /** @var \Kirby\Toolkit\Component $this */
```

Comment:
Wouldn't this (and the following) be rather `\Kirby\From\Field`?

Reply:
You are right! When I wrote the hints I looked up the calling code and saw `$this`, but `Form\Field` extends `Toolkit\Component`. Now fixed. :)